src/config.c

```diff
@@ -8,6 +8,11 @@
 #include "rmutil/strings.h"
 #include "config.h"
 #include "consts.h"
+#include "common.h"
+#include <string.h>
+#include <stdbool.h>
+#include <stdarg.h>
+#include <assert.h>
```

Comment:
@filipecosta90 do we need all of these includes?

Reply:
string.h - strstr
assert.h - assert 
stdbool.h ( not needed after removal of crdt check )
stdarg ( not needed )
removing :)